### PR TITLE
Add E2E testing

### PR DESCRIPTION
**Pull request by floryst**
Adds WebdriverIO end-to-end testing.

Tests added:
- [x] download and render the prostate sample

### CODE REVIEW

**Comment by floryst** [comment]
Windows test seems to be more flaky than I would like...

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - name: Install dependencies
+        run: npm ci
+      - name: Install xvfb
+        if: matrix.os == 'ubuntu-20.04'
+        run: sudo apt-get install xvfb
+      - name: Build for testing
+        run: npm run build
+      - name: Run E2E on chrome (linux)
+        if: matrix.os == 'ubuntu-20.04'
+        run: xvfb-run --server-args="-screen 0, 1400x1000x24" --auto-servernum npm run test:e2e:chrome
```
Comment (PaulHax) [inline .github/workflows/e2e.yml:25]: Interesting!

**Review by PaulHax — APPROVED** [review]
Cool!

Sadly when I `npm run test:e2e:chrome` the test does not work:

```
[0-0] 2023-08-15T21:37:31.326Z INFO webdriver: RESULT {
[0-0]   error: 'no such element',
[0-0]   message: 'no such element: Unable to locate element: {"method":"css selector","selector":"div[data-testid="samples-list"]"}\n' +
[0-0]     '  (Session info: chrome=115.0.5790.170)',
[0-0]   stacktrace: '#0 0x55d71913f4e3 <unknown>\n' +
``` 

Ubuntu OS, Chrome 115

**Comment by floryst** [comment]
Right now the snapshots are taken from github CI. You can look in the `.tmp/` folder to see the diff and actual screenshots. Size and DPR are two possible reasons.

**Comment by floryst** [comment]
We can add more e2e tests when necessary.

**Pull request merged**